bullet_train-super_scaffolding/app/models/scaffolding/completely_concrete/tangible_thing/filter.rb

```diff
@@ -0,0 +1,36 @@
+class Scaffolding::CompletelyConcrete::TangibleThing::Filter < ApplicationFilter
+  def conditions
+    [
+      condition(:text_field_value, :text),
+      condition(:email_field_value, :text),
+      condition(:phone_field_value, :text),
+      condition(:address_value, :text),
+      condition(:text_area_value, :text),
+      condition(:action_text_value, :text),
+
+      condition(:sort_order, :numeric),
+
+      condition(:color_picker_value, :option),
+      condition(:button_value, :option),
+      condition(:boolean_button_value, :option),
+      condition(:multiple_button_values, :option),
+      condition(:option_value, :option),
+      condition(:multiple_option_values, :option),
+      condition(:super_select_value, :option),
+      condition(:multiple_super_select_values, :option),
+
+      condition(:date_field_value, :date),
+      condition(:date_and_time_field_value, :datetime),
+      condition(:created_at, :datetime),
```

Comment:
One thing I would recommend is we add in the clause manipulations for created_at and updated_at generators. The IS_SET and IS_NOT_SET clauses serve no purpose for these fields since they are always present so the scaffolding should probably automatically apply
`.without_clauses([DateWithTimeCondition::CLAUSE_SET, DateWithTimeCondition::CLAUSE_NOT_SET])` for simplicity

Reply:
@ssteffen good point! I just pushed 382ef94 which will use a column's `null: false` information to automatically reject the presence concerned clauses aka SET/NOT_SET.

I think I found a bug in Refine though, so I'll submit a PR shortly.